app-forensics/pebear/pe_bear-0.6.5.2.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="PE-bear reverse engineering tool"
+DESCRIPTION="Portable Executable reversing tool with a friendly GUI"
+HOMEPAGE="https://hshrzd.wordpress.com/pe-bear"
+COPYRIGHT="2022-2023 hasherezade"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/hasherezade/pe-bear/archive/refs/tags/v$portVersion.zip"
+CHECKSUM_SHA256="bbd46e4af231ae5257538e8157a077b2fe5843a5dc18beaaaabbae928bfa1025"
+SOURCE_DIR="pe-bear-$portVersion"
+srcGitSubHash1="e2c08098410c85e5546b77a2785444bbc9e0e21b"
```

Comment:
```suggestion
srcGitRev_2="e2c08098410c85e5546b77a2785444bbc9e0e21b"
```

Reply:
Ok, I'll update the recipe within some hours. Thanks.